scripts/bench.rb

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+# Script to test various configurations that can impact performance of the node
+
+require 'fileutils'
+require 'open3'
+require 'optparse'
+require 'pathname'
+require 'tmpdir'
+require 'toml-rb'
+
+# This is just for capturing the snapshot height
+SNAPSHOT_REGEX = /(?<height>\d+)_.*/.freeze
+
+HEIGHTS_TO_VALIDATE = 400
+
+MINUTE = 60
+HOUR = MINUTE * MINUTE
+
+BENCHMARK_SUITE = [
+  {
+    name: 'baseline',
+    config: {
+      'rocks_db' => {
+        'enable_statistics' => false
+      }
+    },
+    build_command: [
+      'cargo', 'build', '--release'
+    ],
+    import_command: [
+      './target/release/forest',
+      '--config', '%<c>s',
+      '--target-peer-count', '50',
+      '--encrypt-keystore', 'false',
+      '--import-snapshot', '%<s>s',
+      '--halt-after-import'
+    ],
+    validate_command: [
+      './target/release/forest',
+      '--config', '%<c>s',
+      '--target-peer-count', '50',
+      '--encrypt-keystore', 'false',
+      '--import-snapshot', '%<s>s',
+      '--halt-after-import',
+      '--skip-load',
+      '--height', '%<h>s'
+    ]
+  },
+  {
+    name: 'paritydb',
+    config: {},
+    build_command: [
+      'cargo', 'build', '--release',
+      '--no-default-features', '--features', 'forest_fil_cns,paritydb'
+    ],
+    import_command: [
+      './target/release/forest',
+      '--config', '%<c>s',
+      '--target-peer-count', '50',
+      '--encrypt-keystore', 'false',
+      '--import-snapshot', '%<s>s',
+      '--halt-after-import'
+    ],
+    validate_command: [
+      './target/release/forest',
+      '--config', '%<c>s',
+      '--target-peer-count', '50',
+      '--encrypt-keystore', 'false',
+      '--import-snapshot', '%<s>s',
+      '--halt-after-import',
+      '--skip-load',
+      '--height', '%<h>s'
+    ]
+  }
+].freeze
+
+BENCH_PATHS = { tmpdir: Dir.mktmpdir('forest-benchs-') }.freeze
+
+# Provides human readable formatting to Numeric class
+class Numeric
+  def to_bibyte
+    syscall('numfmt', '--to=iec-i', '--suffix=B', '--format=%.2f', to_s)
+  end
+end
+
+def tmp_dir
+  BENCH_PATHS[:tmpdir]
+end
+
+def syscall(*command)
+  stdout, _stderr, status = Open3.capture3(*command)
+  return stdout.chomp if status.success?
+
+  raise "#{command}, #{status}"
```

Comment:
I believe this should work. Rubocop may complain though.
```suggestion
  status.success? ? stdout.chomp : raise "#{command}, #{status}"
```